### PR TITLE
Fix #645: Propagate actual cause with TCShutdownServerException excep…

### LINKED ISSUE #645: Exception swallowed if entity creation fails during passive sync
The code in `ManagedEntityImpl#receiveSyncCreateEntity` creates an exception that does not carry over the root cause.

And the callback on exit in `DistributedObjectServer#start` only logs the exception message.

The root cause should be passed in the `TCShutdownServerException` and the log should contain the full exception stack trace.

### PR DESCRIPTION
**Pull request by ramsai1729**
…tion

### CODE REVIEW

File: tc-messaging/src/main/java/com/tc/object/EntityID.java [hunk 1]
```diff
@@ -84,9 +84,6 @@ public static EntityID readFrom(TCByteBufferInput serialInput) throws IOExceptio
 
   @Override
   public String toString() {
-    return "EntityID{" +
-           "className='" + className + '\'' +
-           ", entityName='" + entityName + '\'' +
-           '}';
+    return className + ":" + entityName;
   }
```
Comment (myronkscott) [inline tc-messaging/src/main/java/com/tc/object/EntityID.java]: I'm not sure I like changing this.

Comment (myronkscott) [inline tc-messaging/src/main/java/com/tc/object/EntityID.java]: grab the classname and entity name if you want this format

Comment (ramsai1729) [inline tc-messaging/src/main/java/com/tc/object/EntityID.java]: not sure I understand this, could you tell me what exactly needs to be changed here?

Comment (myronkscott) [inline tc-messaging/src/main/java/com/tc/object/EntityID.java]: remove this change.  If you want this format in code somewhere just produce it from the component parts.

Comment (ramsai1729) [inline tc-messaging/src/main/java/com/tc/object/EntityID.java]: done

**Pull request merged**